db/data.js

```diff
@@ -158,9 +151,9 @@ const produce = [
   {
     foodName: "Blackcurrants",
     imageURL:
-      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637899/Seasonal/057-organic-food_pikcaf.png",
+      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1666175319/Seasonal/002-blackcurrant_vnquni.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "Blackcurrants are a tart, dark-coloured berry that are at their peak in early to mid-summer. Recent research has suggested that berries - like blackcurrants - can help to prevent the onset of Alzheimers disease. Blackcurrants are also high in vitamin C. When looking for blackcurrants, make sure to pick ones with shiny and unblemished skins that are firm and plump. Blackcurrants will keep for several days in the fridge, and can also be frozen. They can be added raw to fruit salads, or otherwise can be cooked in some sugared water to sweeten them a little. Be careful not to overcook them, or they will lose their flavour. Blackcurrants are great turned into jams, juices and jellies, or when added to crumbles, tarts and other desserts like ice cream. Blackcurrants are fairly easy to grow, and prefer moist to well-drained soil. They thrive in full sun to partial shade, and can be grown in containers too - so long as you have one big enough!",
```

Comment:
Does "Alzheimer's" needs an apostrophe?

Reply:
It does! But I didn't put it in because single apostrophes throw up a warning in my editor, and I vaguely remembered breaking the data formatting on the site using one before LMAO. I'll stick one in and see if it lives, because I think it works on Lamb's Lettuce, so maybe I just made a different error.